bluemira/geometry/parameterisations.py

```diff
@@ -1681,41 +1683,78 @@ def _make_tapered_inner_leg(
             Vertical position of top of limb [m]
         z2:
             Vertical position of bottom of limb [m]
+        r_min:
+            Minimum radius of curvature [m]
         axis:
             [x,y,z] vector normal to plane of parameterisation
 
         Returns
         -------
         CAD Wire of the geometry
         """
-        # Bottom straight section
-        p1 = [x_mid, 0, -z_in]
-        p2 = [x_mid, 0, z2]
-        bot_straight = make_polygon([p2, p1], label="inner_limb_mid_down")
-
-        # Curved taper radius
-        x_t = x_mid - x_in
-        alpha = np.arctan(z_in / (x_t))
-        theta_t = np.pi - 2 * alpha
-        r_taper = z_in / np.sin(theta_t)
+        # Pre-calculations necessary
+        if z_in > z1:
+            raise ValueError(
+                "the straight-curve transition point z_in" "must lie between z1 and z2."
+            )
+        if z_in > abs(z2):
+            raise ValueError(
+                "the straight-curve transition point z_in" "must lie between z1 and z2."
+            )
+        eps = np.finfo(np.pi).resolution
```

Comment:
Can we use EPS from base.constants please?

Reply:
I tried EPS which works on anything whose absolute value is ≤1; but beyond that it breaks:
<pre><font color="#26A269">In [</font><font color="#33D17A"><b>1</b></font><font color="#26A269">]: </font>(tau-EPS) - tau
<font color="#C01C28">Out[</font><font color="#F66151"><b>1</b></font><font color="#C01C28">]: </font>0.0

<font color="#26A269">In [</font><font color="#33D17A"><b>2</b></font><font color="#26A269">]: </font>(tau-EPS)==tau
<font color="#C01C28">Out[</font><font color="#F66151"><b>2</b></font><font color="#C01C28">]: </font>True
</pre>
So I had to use np.finfo(...).resolution.

Reply:
ok good to know, thanks for checking!